src/style.css

```diff
@@ -1 +1,16 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+.field {
+  margin-top: 10px;
+}
+
+.fieldset {
+  margin-top: 20px;
+}
+
+.button {
+  margin-top: 20px;
+}
```

Comment:
no needs to add all of this classes.
You may only apply those to your filedset and form field (label or div)
Example:
```
.fieldset:not(:last-child) {
  margin-bottom: 20px
}

.field:not(:last-child) {
  margin-bottom: 10px
}
```